### DISCUSSION TITLE
Allowing one-way audio only for frigate

### DISCUSSION THREAD

**Discussion by sydneysider123** [blakeblackshear/frigate#7177] [Ask A Question] [answered]
Hi

Just wanted to see if anyone knows whether it is possible to enable one-way audio only? I want to be able to hear the audio from my doorbell without activating the speaker on the doorbell itself.

I have the Amcrest AD410 doorbell and the LED around the button on the doorbell itself changes colour to a dark blue whenever the 'talk' functionality is enabled (i.e. the doorbell speaker is enabled) - which then makes it obvious the stream is being watched (sometimes I would rather just ignore unwanted visitors).

Using the "ffmpeg:rtsp_cam#audio=opus" line currently but removing this line still seems to enable the speaker on the doorbell (for both mse and webrtc modes).

Thanks

**Comment by NickM-27** [comment, marked as answer]
> Using the "ffmpeg:rtsp_cam#audio=opus" line currently but removing this line still seems to enable the speaker on the doorbell (for both mse and webrtc modes).

That's not what is enabling it. Go2rtc has it enabled by default. 

Simply add `#backchannel=0` to the end of the doorbell url to disable it
